packages/nextjs/app/builders/0x4a9A95B6fe3b9416f0c78A8735Aa075c75AF46a4/page.tsx

```diff
@@ -0,0 +1,79 @@
+"use client";
+
+import React from "react";
+import Image from "next/image";
+//Icons
+import { FaGithub } from "react-icons/fa";
+import { RiTwitterXFill } from "react-icons/ri";
+import { SiGmail } from "react-icons/si";
+
+const RohanPersonalPage = () => {
+  return (
+    <div className=" flex justify-center items-center">
+      <div>
+        <div className=" pt-24 lg:pt-[100px]">
+          <div className="flex flex-col-reverse lg:flex-row items-center justify-center gap-x-[50px] md:mx-[550px]">
+            <div className=" w-full">
+              <h1 className=" text-[40px] font-bold md:my-[1rem]">Hey! I&apos;m Rohan👋🏼</h1>
+              <p className=" text-[14px] md:text-lg p-1 text-[#e7e5e5] font-semibold w-full text-justify mb-[6px]">
+                I started as a front-end developer, and I&apos;ve grown into someone who simply enjoys building
+                regardless of the tech stack or language. I enjoy solving complex problems and am eager to dive deeper
+                into the Web3 world.
+              </p>
+              <div className=" flex items-center gap-x-2 -my-3">
+                <p>Address :</p>{" "}
+                <p className="px-[5px] py-[1px] font-bold rounded-[8px] bg-[#ffff] text-black text-[14px]">
+                  0x4a9A95B6fe3b9416f0c78A8735Aa075c75AF46a4
```

Comment:
Maybe a good chance to use Scaffold-ETH 2 Address component? 

https://docs.scaffoldeth.io/components/Address